database/rrddim.c

```diff
@@ -527,6 +527,10 @@ void rrddim_free_custom(RRDSET *st, RRDDIM *rd, int db_rotated)
             debug(D_RRD_CALLS, "Removing dimension '%s'.", rd->name);
             freez((void *)rd->id);
             freez(rd->cache_filename);
+#ifdef ENABLE_DBENGINE
+            freez(rd->state->metric_uuid);
```

Comment:
Why is the metric_uuid being freed for ALLOC and NONE?

Reply:
That's wrong I will fix it